src/aave-v2/SupplyVaultBase.sol

```diff
@@ -85,6 +85,9 @@ abstract contract SupplyVaultBase is ERC4626UpgradeableSafe, OwnableUpgradeable
 
     /// PUBLIC ///
 
+    /// @dev The indexes used by this function might not be up-to-date.
+    ///      As a consequence, public functions (like `maxWithdraw`) could underestimate the withdrawable amount.
```

Comment:
what do you mean by "public function like maxWithdraw" here

Reply:
view* my bad will update